gatsby-node.js

```diff
@@ -42,3 +42,22 @@ exports.createPages = ({ graphql, boundActionCreators }) => {
     )
   })
 }
+
+exports.onCreateNode = ({ node, boundActionCreators, getNode }) => {
+  const { createNodeField } = boundActionCreators
```

Comment:
Perhaps use https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-source-filesystem/src/create-file-path.js?

Reply:
@KyleAMathews Are you referring to https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-source-filesystem/src/create-file-path.js#L35-L46, as a suggestion for what should be implemented here?

Reply:
No use it directly. The function is exported from gatsby-source-filesystem and intended to be used for creating slugs from files.